doc/template.html

```diff
@@ -45,5 +45,6 @@
   <script src="assets/sh_main.js"></script>
   <script src="assets/sh_javascript.min.js"></script>
   <script>highlight(undefined, undefined, 'pre');</script>
+  __TRACKING__
```

Comment:
@jbergstroem sorry this slipped. I just pushed an update using a comment as you've suggested, PTAL.